interface/main/messages/js/reminder_appts.js

```diff
@@ -0,0 +1,459 @@
+                                                       var new_date = moment($(this).val(),'days');
+                                                       now = dolv.add($(this).val(),'days').format('YYYY-MM-DD');
+                                                       $("#datepicker2").val(now);
+                                                       });
+                  $("[name='tabs']").click(function() {
+                                           $("[name='tabs']").removeClass('no_bot_border');
+                                           $(this).addClass('no_bot_border');
+                                           show_just = this.id.match(/tab_(.*)$/);
+                                           var dcolor = show_just[1];
+                                           if (show_just[1] == 'ALL') {
+                                           $(".ALL").removeClass('nodisplay');
+                                           $("[name='visible_cols']").removeClass('fa-toggle-off').addClass('fa-toggle-on');
+                                           $('#reminder_wrap').removeClass('whitish')
+                                           .removeClass('reddish')
+                                           .removeClass('greenish')
+                                           .removeClass('yellowish')
+                                           .addClass('whitish');
+                                           } else {
+                                           $(".ALL").addClass('nodisplay');
+                                           $("."+show_just[1]).removeClass('nodisplay');
+                                           $('#reminder_wrap').removeClass('whitish')
+                                           .removeClass('reddish')
+                                           .removeClass('greenish')
+                                           .removeClass('yellowish')
+                                           .addClass(dcolor);
+                                           }
+                                           //reminder_wrap div needs to change color too.
+                                           });
+                  $(".update").on('change',function(e) {
+                                  var formData = $("form#save_prefs").serialize();
+                                  var url = "save.php";
+                                  top.restoreSession();
+                                  $.ajax({
+                                         type       	: 'POST',
+                                         url        	:  url,
+                                         data     	:  formData,
+                                         action      : 'save_prefs'
+                                         }).done(function(result) {
+                                                 $("#div_response").html('<span style="color:red;">Preferences updated successfully.</span>');
```

Comment:
Will need to translate this (this is actually straightforward; basically make a javascript variable in the php script that calls this script, which contains the translated constant and then use the variable here).